rosdep/base.yaml

```
@@ -1336,6 +1336,18 @@ gitk:
   debian: [gitk]
   fedora: [gitk]
   ubuntu: [gitk]
+git-lfs:
+  alpine: [git-lfs]
+  arch: [git-lfs]
+  debian: [git-lfs]
```

CI is reporting that this package is not available for `stretch`, which looks to be correct.

```suggestion
  debian:
    '*': [git-lfs]
    stretch: null
```

---

Thank you so much!
This is exactly the writing style I was looking for.